app/models/location_facility.rb

```diff
@@ -0,0 +1,4 @@
+class LocationFacility < ApplicationRecord
+  belongs_to :location
+  belongs_to :facility
```

Comment:
Using has_and_belongs_to_many for Facility and location can avoid creating a dummy model here.